operations/service.go

```diff
@@ -176,3 +176,86 @@ func signalListener(ctx context.Context, trigger context.CancelFunc) {
 
 	trigger()
 }
+
+func setupDepot(conf model.CertDepotConfig) (depot.Depot, error) {
```

Comment:
feels like we should move from here down into certDepot? (and maybe the config type?) and also test it? 

Also I think that model.CertDepotConfig should hold the CA cert/key so that you can bootstrap from a known ca?